flytekit/common/tasks/presto_task.py

```diff
@@ -122,6 +127,21 @@ def __call__(self, *args, **kwargs):
             *args, **kwargs
         )
 
+    # Override method in order to set the implicit inputs
+    def _python_std_input_map_to_literal_map(self, inputs):
+        """
```

Comment:
Can you explain what the stuff in the __call__ function does and why it's not enough please, for me and for posterity to understand?

Reply:
ordinarily we use `__call__` to create an SDK node, which is formerly the only the way a task was transformed for execution.

but when we call `launch` on a task to trigger a single task execution [here](https://github.com/lyft/flytekit/blob/master/flytekit/common/mixins/launchable.py#L32) we don't use this code path and the implicit inputs are never added

Reply:
Should we "unify" the setting of those implicit inputs between those two call paths? maybe make __call__ call this function or something?

Reply:
`__call__` only creates bindings for inputs, rather than converting them to sdk types but good point i can refactor the behavior to a shared function